Multisig/Info.plist

```diff
@@ -63,6 +63,10 @@
 	<string>$(HELP_CENTER_URL)</string>
 	<key>INFURA_API_KEY</key>
 	<string>$(INFURA_API_KEY)</string>
+	<key>INTERCOM_APP_ID</key>
```

Comment:
I see these variables here, it's good! Can you also add the example values to the `Config.Example.xcconfig` so that we don't forget to set these variables?

Reply:
we also need to provide these variables in our CI environment, I'll help you with that